### PR TITLE
Issue 15 - delete reminders

### LINKED ISSUE #15: Remove Reminder
User can remove/delete a reminder.

* Users should be able to remove a reminder from both the list of reminders or from an individual reminder’s page.

* If a reminder is removed from the individual reminder’s page, then the application should reroute them back to the `'/reminders'` route.

### PR DESCRIPTION
**Pull request by hilarylewis92**
## Purpose

Delete reminders from reminders list and individual reminder

## Approach

We added a Controller to handle the delete action. It then reroutes to /reminders. 

### Learning

Google and looked at the Ember docs for deleting and rerouting

### Test coverage 

We wrote tests to see if the delete button appears, and on click, that it disappears. 

Closes #15 and #13.

### CODE REVIEW

**Comment by hilarylewis92** [comment]
We need to fix the merge conflict.

**Comment by hilarylewis92** [comment]
@martensonbj @nfosterky @Tman22 

Fixed merge conflicts. Good to go!

File: app/templates/reminders/reminder.hbs [hunk 1]
```diff
@@ -8,6 +8,9 @@
   <p class="spec-selected-notes">
     {{model.notes}}
   </p>
+  <button class='spec-delete' {{action 'delete' model on="click"}}>
```
Comment (martensonbj) [inline app/templates/reminders/reminder.hbs]: `click` is the default action so you don't need the `on="click"` bit here

File: tests/unit/controllers/reminders-test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
```
Comment (martensonbj) [inline tests/unit/controllers/reminders-test.js]: If we're not going to use these test files lets get rid of them

File: tests/unit/controllers/reminders/reminder-test.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
```
Comment (martensonbj) [inline tests/unit/controllers/reminders/reminder-test.js]: Delete unused test files

**Comment by hilarylewis92** [comment]
Change requests should be resolved.

**Pull request merged**